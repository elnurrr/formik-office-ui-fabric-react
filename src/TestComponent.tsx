import React from 'react'

export const TestComponent = ({ onClick }: { onClick?: any }) => (
  <div>
    test
    <button onClick={onClick}>+</button>
  </div>
)
