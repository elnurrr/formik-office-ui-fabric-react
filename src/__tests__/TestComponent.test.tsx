import * as React from 'react'
import renderer from 'react-test-renderer'
import { TestComponent } from '../TestComponent'

test('TestComponent renders correctly', () => {
  const component = renderer.create(<TestComponent />)

  expect(component.toJSON()).toMatchSnapshot()
})
