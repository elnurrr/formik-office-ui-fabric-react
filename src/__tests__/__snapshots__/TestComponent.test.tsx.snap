// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TestComponent renders correctly 1`] = `
<div>
  test
  <button>
    +
  </button>
</div>
`;
